flux_led/utils.py

```diff
@@ -200,14 +261,7 @@ def rgbww_brightness(
             (255 - cw_brightness) * change_brightness_pct + cw_brightness
         )
 
-    r, g, b = colorsys.hsv_to_rgb(h, s, color_brightness / 255)
-    return (
-        round(r * 255),
-        round(g * 255),
-        round(b * 255),
-        ww_brightness,
-        cw_brightness,
-    )
+    return color_brightness, cw_brightness, ww_brightness
 
```

Comment:
This needs to return and RGB.  HA determine the color brightness on the front end.

Reply:
5177a2f86cc16a83c37806dd981b2a990bac4439